File: JavaScript Fundamentals/FunctionsHomework/01EnglishDigit/English digit.js
function digitAsWord() {
	var input = document.getElementById("input").value;

	switch(input[input.length - 1]) {
		case '0': document.getElementById("result").value = 'zero'; break;
		case '1': document.getElementById("result").value = 'one'; break;
		case '2': document.getElementById("result").value = 'two'; break;
		case '3': document.getElementById("result").value = 'three'; break;
		case '4': document.getElementById("result").value = 'four'; break;
		case '5': document.getElementById("result").value = 'five'; break;
		case '6': document.getElementById("result").value = 'six'; break;
		case '7': document.getElementById("result").value = 'seven'; break;
		case '8': document.getElementById("result").value = 'eight'; break;
		case '9': document.getElementById("result").value = 'nine'; break;
		default: document.getElementById("result").value = 'Not a valid input'; break;
	}
}

File: C# Part2/MethodsHomework/EnglishDigit/EnglishDigit.cs
//Problem 3. English digit
//Write a method that returns the last digit of given integer as an English word.
using System;
class EnglishDigit
{
    static void EnglishWord(int number)
    {
        int lastdigit = number % 10;
        
        Console.Write("The last digit is: ");
        switch (lastdigit)
        {
            case 0: Console.WriteLine("Zero"); break;
            case 1: Console.WriteLine("One"); break;
            case 2: Console.WriteLine("Two"); break;
            case 3: Console.WriteLine("Three"); break;
            case 4: Console.WriteLine("Four"); break;
            case 5: Console.WriteLine("Five"); break;
            case 6: Console.WriteLine("Six"); break;
            case 7: Console.WriteLine("Seven"); break;
            case 8: Console.WriteLine("Eight"); break;
            case 9: Console.WriteLine("Nine"); break;
        }
    }
    static void Main()
    {
        Console.Write("Enter a number: ");
        int number = int.Parse(Console.ReadLine());

        EnglishWord(number);
    }
}